libraries/ui/src/main/java/androidx/media3/ui/PlayerControlView.java

```diff
@@ -193,7 +193,8 @@
 public class PlayerControlView extends FrameLayout {
 
   static {
-    MediaLibraryInfo.registerModule("media3.ui");
+    MediaLibraryInfo.registerModule("media3.ui"); // copybara:media3-only
+    // copybara:exo-only MediaLibraryInfo.registerModule("goog.exo.ui");
```

Comment:
Nice catch.

I don't remember removing these comments.

Reply:
Its okay. It was not you actually. These comments I should not have readded here. But thank you for noticing!